libcontainer/nsenter/ipc.h

```diff
@@ -2,6 +2,11 @@
 #define NSENTER_IPC_H
 
 int receive_fd(int sockfd);
-void send_fd(int sockfd, int fd);
```

Comment:
Ideally this should be a separate patch. Same for receive_fd which is converted in the same way in the commit claiming to only move the code around.

So maybe
 - move the code around in the first commit
 - do the changes like this in the second commit
 - introduce the new feature in the last commit.